deeposlandia/datasets/tanzania.py

```diff
@@ -195,15 +330,24 @@ class method genericity
         image_list = os.listdir(os.path.join(input_dir, "images"))
         image_list_longname = [os.path.join(input_dir, "images", l)
                                for l in image_list
-                               if not l.startswith('.')][:nb_images]
+                               if not l.startswith('.')]
+        nb_image_files = len(image_list_longname)
 
         logger.info("Getting %s images to preprocess..."
-                    % len(image_list_longname))
+                    % nb_image_files)
         logger.info(image_list_longname)
-        with Pool() as p:
-            self.image_info = p.starmap(self._preprocess,
-                                        [(x, output_dir, labelling)
-                                         for x in image_list_longname])
+        if labelling:
+            nb_tile_per_image = int(nb_images/nb_image_files)
+            with Pool(processes=3) as p:
```

Comment:
maybe the number of process can be parameterized? Either it's a parameter of the CLI or a new variable in a config file or a environment variable name.

Reply:
Ok, I will put it in `config.ini` file (and update `config.ini.sample` accordingly), it makes sense to manage it depending on the running system.